.circleci/config.yml

```diff
@@ -384,6 +384,51 @@ jobs:
           command: make test
           working_directory: l2geth
 
+  devnet:
+    machine:
+      image: ubuntu-2004:202201-01
+      docker_layer_caching: true
+    environment:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - checkout
+      - run:
+          name: Check if we should run
+          command: |
+            CHANGED=$(bash ./ops/docker/ci-builder/check-changed.sh "(contracts-bedrock|op-bindings|op-batcher|op-node|op-proposer|ops-bedrock)/")
+            if [[ "$CHANGED" = "FALSE" ]]; then
+              circleci step halt
+            fi
+      - run:
+          name: foundryup
```

Comment:
Use a builder that has foundry? This seems to be the only reference to foundry.

Reply:
I wasn't sure if we can run docker in docker so i am installing foundry this way

Reply:
docker in docker would be a bit nice to have the same foundry version, but this seems fine then

Reply:
I've run docker in docker on Circle before, but it's a bit complicated. I'm also good with merging as-is, we can refactor later.